### ISSUE TITLE
Add rotating banner of case study  quotes and testimonials 

### ISSUE THREAD

**Issue by nainaz** [knative/community#1417]
Description
I would like to request adding a rotating banner of logos of case studies using knative project to the knative.dev site. This would be a great way to:

Showcase the success stories of people who have used the project
Provide social proof to potential users
Encourage more people to use the project

Proposed Solution
The rotating banner would be implemented as a simple carousel of images. The carousel would rotate automatically at a fixed interval.

Additional Information
Case studies are available in https://github.com/knative/docs/tree/main/docs/about/case-studies
Testimonials are available in https://github.com/knative/docs/blob/main/docs/about/testimonials.md
Timeline
I would like to implement this feature within the next two weeks.

Questions?
Please let me know if you have any questions.

Here are some additional details you can add to your issue:

How many logos do you plan to include in the rotating banner?
How will you select the logos to include?
Will you include a link to each case study's website?
What is the expected impact of this feature on the project?
I hope this helps!

**Comment by aliok** [comment]
The comment in a similar issue can be used to get started: https://github.com/knative/community/issues/1416#issuecomment-1678083055

**Comment by aliok** [comment]
@knative/ux-wg-leads do you think the rotating case studies make sense? How can we show the 5 case studies we have on the home page?

My opinion is that we don't need to show all of them there; we should just continue showing 3. 
They're shown in the case studies page anyway: https://knative.dev/development/about/case-studies/ 

And, we have the rotating banners already: https://knative.dev/development/

Might look weird if too many things are rotating/sliding in the webpage.

What do you think?

**Comment by Leo6Leo** [comment]
I agree with @aliok . I think if there are too much things moving/rotating on the web page, it will distract the users. 

I like that @nainaz suggested to show users that we have few end-user case studies, and it is a great showcase of the success stories of people who are using/have used Knative!

So I am here proposing 2 alternative solutions:
1. we keep the form we have right now on the website (showing only 3 case study), and each time when we refresh the page, it will show 3 random case studies we have. But this might take some time to implement.
2. We display all the case studies, and make the section horizontally scrollable. Something similar to [this](https://www.apple.com/ca/store#:~:text=The%20latest.%20Take%20a%20look%20at%20what%E2%80%99s%20new%20right%20now.)

**Comment by nainaz** [comment]
Agree!
we should wait
Thank you,
-N

**Comment by aliok** [comment]
https://github.com/knative/ux/issues/146 would be a nice solution to this issue. Thanks @asr2003
